File: MedLabO.Models/Requests/Termin/TerminOdobravanjeRequest.cs

namespace MedLabO.Models.Requests.Termin
{
    public class TerminOdobravanjeRequest
    {
        public Guid TerminID { get; set; }
        public string? Odgovor { get; set; }
        public bool Status { get; set; } = false;
    }
}

File: MedLabO UI/medlabo_desktop/lib/models/termin/termin_odobravanje_request.dart
import 'package:json_annotation/json_annotation.dart';

part 'termin_odobravanje_request.g.dart';

@JsonSerializable()
class TerminOdobravanjeRequest {
  String? terminID;
  String? odgovor;
  bool? status;

  TerminOdobravanjeRequest({
    this.terminID,
    this.odgovor,
    this.status,
  });

  factory TerminOdobravanjeRequest.fromJson(Map<String, dynamic> json) =>
      _$TerminOdobravanjeRequestFromJson(json);

  Map<String, dynamic> toJson() => _$TerminOdobravanjeRequestToJson(this);
}
